### PR TITLE
Higher test Cov

### PR DESCRIPTION
**Pull request by ethanleifer**
Up testing cov from 50% to 70%

### CODE REVIEW

File: packages/indexer/indexer/process.py [hunk 1]
```diff
@@ -40,9 +53,42 @@ async def process_tx(
...
-            await upsert_raw_txs(pool, {height: txs_data}, chain.chain_id)
+            await upsert_raw_txs(db, {height: txs_data}, chain.chain_id)
             return True
-
     except Exception as e:
         print(f"upsert_txs error {repr(e)} - {height}")
         return False
+
+
+class DbNotificationHandler:
```
Comment (ethanleifer) [inline packages/indexer/indexer/process.py:63]: Interesting change

File: packages/indexer/tests/db_test.py [hunk 2]
```diff
@@ -24,77 +35,133 @@ async def mock_pool():
...
 ):
-    await create_tables(mock_pool)
-    for block in raw_blocks:
-        await upsert_raw_blocks(mock_pool, block)
+    await create_tables(mock_db)
+    for block in raw_blocks["block"][: len(raw_blocks) // 2]:
+        await upsert_raw_blocks(mock_db, block)
+
+    # valid process block
+    for height, _chain_id, block, _parsed_at in raw_blocks[
```
Comment (ethanleifer) [inline packages/indexer/tests/db_test.py:128]: Split testing data into two sections to test two different implementations

Then test incorrect data

File: packages/indexer/tests/db_test.py [hunk 3]
```diff
@@ -103,6 +170,75 @@ async def test_upsert_raw_blocks(
...
+            f"""
+            select * from {mock_db.schema}.raw_txs
+            """
+        )
+
+        results = await conn.fetch(f"select * from {mock_db.schema}.txs")
+
+    assert len(results) == len(flattened_raw_txs)
+
+    while len(mock_notification_handler.notifications) < len(flattened_raw_txs):
```
Comment (ethanleifer) [inline packages/indexer/tests/db_test.py:230]: wait to see if number of notifications match number of txs, then close notification handler

File: packages/indexer/tests/parser_test.py [hunk 4]
```diff
@@ -63,24 +66,9 @@ def test_parse_individual_log_event():
     }
 
 
-def test_parse_tx_logs():
+def test_parse_tx_logs(mock_log_data):
```
Comment (ethanleifer) [inline packages/indexer/tests/parser_test.py:69]: erroring test

mock_log_data is some jackal data. Have 0 ideas where it is coming from

File: packages/indexer/tests/db_test.py (hunk 2 above)
Comment (zackgow) [reply packages/indexer/tests/db_test.py:128]: Good

File: packages/indexer/tests/parser_test.py (hunk 4 above)
Comment (zackgow) [reply packages/indexer/tests/parser_test.py:69]: Explain?

Comment (ethanleifer) [reply packages/indexer/tests/parser_test.py:69]: mock log data isn't what it should be

**Pull request merged**